src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -408,13 +408,13 @@ private ImmutableArray<DiagnosticId> GetFixableDiagnosticIds(CodeFixProvider fix
             {
                 return extensionManager.PerformFunction(
                     fixer,
-                    () => ImmutableInterlocked.GetOrAdd(ref _fixerToFixableIdsMap, fixer, f => f.FixableDiagnosticIds),
+                    () => ImmutableInterlocked.GetOrAdd(ref _fixerToFixableIdsMap, fixer, GetAndTestFixableDiagnosticIds),
                     defaultValue: ImmutableArray<DiagnosticId>.Empty);
             }
 
             try
             {
-                return ImmutableInterlocked.GetOrAdd(ref _fixerToFixableIdsMap, fixer, f => f.FixableDiagnosticIds);
+                return ImmutableInterlocked.GetOrAdd(ref _fixerToFixableIdsMap, fixer, GetAndTestFixableDiagnosticIds);
```

Comment:
:bulb: Consider using `f => GetAndTestFixableDiagnosticIds(f)` instead of just `GetAndTestFixableDiagnosticIds` so the delegate is cached.